RELEASE-NOTES.md

```diff
@@ -1,5 +1,107 @@
 # Release notes for Reactive Streams
 
+---
+
+# Version 1.0.2-RC1 released on 2017-12-01
+
+## Announcement:
+
+This update to `Reactive Streams` brings the following improvements over `1.0.1`.
+
+## Highlights:
+
+- Specification
+  + Glossary term added for `Thread-safe`
+  + No breaking/semantical changes
+  + Rule [clarifications](#specification-clarifications-1.0.2-RC1)
```

Comment:
The `.` in these links are removed by GitHub's markdown and the URL becomes:

`RELEASE-NOTES.md#specification-clarifications-102-rc1`

Reply:
Nice catch, thanks @akarnokd!